torch/csrc/jit/codegen/cuda/utils.cpp

```diff
@@ -37,6 +37,8 @@ auto parseDebugDumpOptions() {
         options_map[DebugDumpOption::CudaKernel] = true;
       } else if (token == "cuda_full") {
         options_map[DebugDumpOption::CudaFull] = true;
+      } else if (token == "launch_param") {
```

Comment:
I agree this is very useful, but should it be included as an option of `PYTORCH_NVFUSER_DUMP`? The other options are for dumping IR states. Perhaps, a different environment variable, like `PYTORCH_NVFUSER_TRACE` , may be more suitable.

In any case, the error message at line 48 should be updated with `launch_param` if it's included in this option set.

Reply:
Updated the error message. I'm hesitant to introduce a new env variable in this case. We could revisit later especially if we have a good list of functionalities that would go into the `PYTORCH_NVFUSER_TRACE` category.